index.js

```diff
@@ -76,11 +76,11 @@
 				observeRefreshSelector = ".ghx-column-header-group";
 			} else if (this.isBoardTypeNextGen) {
 				headerColsSelector =
-					'[data-test-id="platform-board-kit.common.ui.column-header.header.column-header-container"]';
+					'[data-testid="platform-board-kit.common.ui.column-header.header.column-header-container"]';
 				laneColsSelector =
 					'[data-component-selector="platform-board-kit.ui.column.draggable-column"]';
 				estimateSelector =
-					'[data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
+					'[data-testid="platform-card.common.ui.estimate.badge"]';
```

Comment:
@arielscarpinelli Hi, first of all thanks for the PR :)

i tested this out the Jira cloud version, and the deleted one still seems to be working there. Maybe the best solution would be to just include both:

```
estimateSelector =
					'[data-testid="platform-card.common.ui.estimate.badge"], [data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
```

Reply:
```suggestion
					'[data-testid="platform-card.common.ui.estimate.badge"], [data-testid="software-board.common.fields.estimate-field.static.estimate-wrapper"]';
```